### ISSUE TITLE
Grafana service fails to install on fresh cluster

### ISSUE THREAD

**Issue by ymesika** [istio/istio#6238]
When enabling grafana in `values.yaml` and trying to install it together with Istio, Helm throws the following error because the CRDs aren't there yet:

```sh
Error: apiVersion "authentication.istio.io/v1alpha1" in istio/charts/grafana/templates/service.yaml is not available
```

**Comment by ymesika** [comment]
Might be relevant:
https://github.com/kubernetes/helm/issues/2994
https://github.com/kubernetes/charts/pull/2369

**Comment by ymesika** [comment]
Fixed by #6910
